config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_alloydbinstances.alloydb.cnrm.cloud.google.com.yaml

```diff
@@ -180,6 +180,29 @@ spec:
                     description: The number of CPU's in the VM instance.
                     type: integer
                 type: object
+              pscInstanceConfig:
+                description: Configuration for Private Service Connect (PSC) for the
+                  instance.
+                properties:
+                  allowedConsumerProjects:
```

Comment:
This fields looks like a reference field referencing Project resources.

Here is the guide doc: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/README.ConfigureResourceReferences.md